test/e2e/save_test.go

```diff
@@ -226,13 +226,17 @@ default-docker:
 	})
 
 	It("podman save --multi-image-archive (untagged images)", func() {
-		// Refer to images via ID instead of tag.
-		session := podmanTest.Podman([]string{"images", "--format", "{{.ID}}"})
+		// #14468: to make execution time more predictable, save at
+		// most three images and sort them by size.
+		session := podmanTest.Podman([]string{"images", "--sort", "size", "--format", "{{.ID}}"})
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
 		ids := session.OutputToStringArray()
 
 		Expect(len(ids)).To(BeNumerically(">", 1), "We need to have *some* images to save")
+		if len(ids) > 3 {
+			ids = ids[:2]
+		}
```

Comment:
`[:2]` means that only the first two elements are used, and not three. I guess it does not matter for the test purpose but looks like this is not what you want based on the commit description.

Reply:
Thank you for catching.  I updated the PR.